chain/tendermint/events.go

```diff
@@ -1,24 +1,22 @@
 package tendermint
 
 import (
-	"bytes"
-
 	abcitypes "github.com/tendermint/tendermint/abci/types"
 )
 
 // AttributeValue returns an event attribute value given the eventType and attribute key tuple.
 // In the event of duplicate types and keys, returns the first attribute value found.
 // If not found, ok is false.
```

Comment:
If it is correct to drop the `ok` return value, then this line can be removed.

But without specific domain knowledge about the events, I don't know whether that is accurate. If it is illegal to have an empty value, then it seems fine to drop `ok` -- maybe the comments should reflect that.

But if it is possible that an attribute value that is an empty string, then we should  keep `ok`.

Reply:
Went back and for on this. With strings, the zero value is safe from null dereferences, so I figured callers could check for an empty string if they so cared. I will probably add back the `ok` as you suggested. The `tx.Events` is a wildly loose data structure, so perfectly reasonable to not find something.